lib/typing/columns/columns.go

```diff
@@ -73,6 +72,11 @@ func (c *Column) ToLowerName() {
 	c.name = strings.ToLower(c.name)
 }
 
+// SetPrimaryKey is used for tests only.
+func (c *Column) SetPrimaryKey(primaryKey bool) {
```

Comment:
Maybe we could name this `SetPrimaryKeyForTest` so it's extra clear that we shouldn't use it outside of tests?

Reply:
sgtm